File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for (int i = 0; i < commands.length; i++){
            int a = commands[i][0];
            int a2 = a - 1;
            int b = commands[i][1];
            int c = commands[i][2];
            int[] s = new int[b - a + 1];
            for (int j = 0; j < b - a + 1; j++) {
                s[j] = array[a2];
                a2++;
            }
            Arrays.sort(s);
            answer[i] = s[c - 1];
        }
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for i in range(len(commands)):
        k_nums = []
        
        for num in range(commands[i][0] - 1, commands[i][1]):
            k_nums.append(array[num])
        k_nums.sort()
        print(k_nums)
        answer.append(k_nums.pop(commands[i][2] - 1))
    
    return answer